packages/openneuro-app/src/scripts/users/dataset-card.tsx

```diff
@@ -87,6 +89,14 @@ export const DatasetCard: React.FC<DatasetCardProps> = (
       datasetSize = `${sizeInBytes} bytes`
     }
   }
+
+  const year = getYear(parseISO(dataset.created))
+  const authors = dataset.latestSnapshot.description?.Authors
+    ? dataset.latestSnapshot.description.Authors.join(" and ")
```

Comment:
For display of this, I think we should make two changes:
1. List authors comma separated with only one "and" (preceding the final author's name).
1. If there's many authors, limit the number and replace the list with ... or "et al.".